code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -95,6 +95,23 @@
 	mag_display_ammo = TRUE
 	empty_indicator = TRUE
 
+/obj/item/gun/ballistic/automatic/plastikov
+	name = "\improper PP-95 SMG"
+	desc = "An ancient 9x19mm submachine gun pattern updated and simplified to lower costs, though perhaps simplified too much."
+	icon_state = "plastikov"
+	icon_state = "plastikov"
+	mag_type = /obj/item/ammo_box/magazine/plastikov9mm
+	w_class = WEIGHT_CLASS_NORMAL
+	fire_delay = 2
+	burst_size = 5
+	spread = 30
```

Comment:
Did they put a toilet pipe on this thing because it's gonna be spraying the walls with shit

Reply:
yes